crates/llvm-context/src/polkavm/context/debug_info.rs

```diff
@@ -1,14 +1,52 @@
 //! The LLVM debug information.
 
+use std::cell::RefCell;
+use std::rc::Rc;
+
 use inkwell::debug_info::AsDIScope;
-use num::Zero;
+use inkwell::debug_info::DIScope;
+
+/// Debug info scope stack
+#[derive(Clone, Debug, PartialEq, Eq)]
+pub struct ScopeStack<'ctx> {
+    stack: Vec<Rc<DIScope<'ctx>>>,
```

Comment:
Shouldn't be necessary to wrap the the DIScope into an `Rc` here

Reply:
Fixed.